scripts/upgrade-via-npm.ps1

```
@@ -18,40 +26,102 @@ param(
+        if ($jobState -ne 'Running') { break }
+        Start-Sleep -Milliseconds 500
+    }
+
+    # 输出命令执行结果
+    $output = ""
+    try {
+        $output = Receive-Job -Job $job -ErrorAction SilentlyContinue
+        if ($output) {
+            $output | ForEach-Object { Write-Host "    $_" -ForegroundColor Gray }
+        }
+    } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)